CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: MIT
 
-cmake_minimum_required(VERSION 3.22.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.16.0 FATAL_ERROR)
```

Comment:
I see in the description that this was purposeful, but this was deliberately raised from 3.8.0 to 3.22.0 here: https://github.com/oneapi-src/unified-runtime/pull/147/files

It's not clear to me why that PR needed to bump the CMake version, but is 3.16.0 definitely high enough?

Reply:
I'm not sure why the version was changed. From my perspective the version I have with ubuntu 20.04 is 3.16 which I think is default. I believe 3.22 is the new default version shipped with 22.04. I'm happy to update my version if it's worthwhile @pbalcer?

Reply:
CMake 3.22.0 is very new, I don't think we should be this aggresive with requiring such a new CMake version with `FATAL_ERROR`. I think we should be matching the CMake version as https://github.com/intel/llvm requires since we are a direct dependency of that project.

Reply:
No, I agree we should roll back this change. @PatKamin was dealing with a very annoying problem on Windows that was solvable by using a very new cmake feature (which we realized was not really viable), but ultimately he managed to address the issue differently.

Reply:
Sounds good with me. I'll rollback to version `3.14` to match intel's llvm/sycl version